### ISSUE TITLE
A tag for "Free" modules

### ISSUE THREAD

**Issue by TheOriginalSoni** [AdventureLookup/AdventureLookup#384]
This website is good, but having a tag for "Free" (or even "Free"/"Donate"/"Paid") would easily turn it from a good resource to have, to a Holy Grail everyone would reference.

A reasonably large chunk of the playerbase and DMs first search for free modules and adventures before turning to paid versions. By adding the tag, it guarantees they check AdventureLookup first.

Obviously real time updating of changing prices is not easy or feasible, but just a tag for "Free" would be a massive QoL improvement for a much lesser effort.

**Comment by cmfcmf** [comment]
Hi @TheOriginalSoni,

thank you for the suggestion. A price or "free" field has been requested before, see #152, #225, #123, [reddit 1](https://www.reddit.com/r/AdventureLookup/comments/6ug23a/free_download_as_a_search_setting/), [reddit 2](https://www.reddit.com/r/AdventureLookup/comments/gz2m56/please_add_a_free_tag/) (I see that this where you are coming from).

These issues and discussions list multiple reasons why such a field was rejected in the past, and also have some comments by Matt Colville on the topic.

**Comment by TheOriginalSoni** [comment]
If a functionality is requested so many times, maybe a comment from 3 years ago should be revisited?

Either way, a free tag would be amazing from User POV. I understand if you don't want to add it, it just means it's just not a useful enough website for me to use.

**Comment by pepper-jk** [comment]
I would agree that this sould be revisited. After skimming over the older issues and concerns from @MattColville I'd say he was more worried about the high maintenance of a hard coded "price" field.

What @TheOriginalSoni is proposing is a field with the categories "Free"/"Not Free" or "Free"/"Donate"/"Paid", which should not chance with high frequency if at all. This would already be enough information for most users and it would not be necessary to add a hard coded price to an adventure. I'd say this **is not the same issue** as the ones @cmfcmf linked above.

Additionally I'd like to propose a second field called **"license"**. This way one could filter for **CC** and **OGL** content as well. (If you want I can open a separate issue for this though.)

p.s.: Regarding the price: shouldn't it be possible to just grab the prices from the provided link? Most adventures will be hosted on DMsGuild or DriveThruRPG right? Should be easy enough the grab the price from there dynamically. (Just a thought.)

**Comment by JohnnyFlash** [comment]
@pepper-jk You should add the license idea as another issue.  It's a good suggestion.

The real question is why does it matter if it is free?  Is 99 cents **that** different from free?

Isn't the real goal to find the best adventure?

**Comment by pepper-jk** [comment]
> @pepper-jk You should add the license idea as another issue. It's a good suggestion.

I will do so tomorrow. It's almost 3 am here. :D
I want to make a more detailed proposal when opening the issue.

> The real question is why does it matter if it is free? Is 99 cents **that** different from free?
>
> Isn't the real goal to find the best adventure?

Although I agree that the goal should be to find the best adventure, I feel this is a pretty subjective view on the matter and not everyone will agree with this. And isn't the goal to improve the site for the whole community?

If we are talking opinions, I personally am more comfortable to donate after I have ran or played the module and liked it.

I was just trying to support @TheOriginalSoni as I don't think the issue should be dismissed because similar, but not identical, issue were in the past.

**Comment by TheOriginalSoni** [comment]
> The real question is why does it matter if it is free? Is 99 cents **that** different from free?
> 
> Isn't the real goal to find the best adventure?

Is it? I'd say "finding adventures that suit DMs" to be more relevant than that. And a significant number of those who search end up considering "Is it free" for multiple reasons. 

Not wanting to add the tag is not the same as "The tag is useless", just that you don't want to see the usecase. Multiple people asking for it, shows clearly that it's a useful search criterion
